curriculum/challenges/english/24-b1-english-for-developers/learn-how-to-use-modal-verbs/67d6a570cc0f095ab2c12a5b.md

```diff
@@ -31,7 +31,7 @@ This noun refers to how well something works or operates, such as software speed
 
 - `The new update improved the app's performance.` - The app now works better and faster.
 
-- `We need to test the website's performance on different devices.` - We should check if the website works well on all devices.
+- `We need to test the website's performance on different devices.` - They should check if the website works well on all devices.
```

Comment:
I thought `we` in the examples should be explained as `you`. Why sometimes it's `they`? @nieldakarla